cfgov/unprocessed/js/organisms/VideoPlayer.js

```diff
@@ -70,8 +75,14 @@ function VideoPlayer( element ) {
 
   /**
    * Event handler for when the video is ready.
+   * @param {Object} event -
+   *   Event object containing target to video player instance.
    */
-  function _videoPlayerReadyHandler() {
+  function _videoPlayerReadyHandler( event ) {
+    // Add duration timestamp to video.
+    const player = event.target;
+    _durationDom.innerHTML = formatTimestamp( player.getDuration() );
+    _durationDom.classList.add( 'duration-loaded' );
```

Comment:
Is there an advantage to using `duration-loaded` instead of just toggling it with [`u-hidden`](https://github.com/cfpb/consumerfinance.gov/blob/ad6137e6d7011895f8d7dd138255f45b2b8ceea2/cfgov/unprocessed/js/organisms/VideoPlayer.js#L90-L91)?

Reply:
Nah. Updated in https://github.com/cfpb/consumerfinance.gov/pull/6267/commits/8fa1b3dd4e47337e2002c1fec103197efbbb9f10